subprojects/file-watching/src/main/java/org/gradle/internal/watch/vfs/impl/WatchingVirtualFileSystem.java

```diff
@@ -130,12 +135,15 @@ public FileSystemWatchingStatistics getStatistics() {
                                           }
                                       }
                     );
+                    if (verboseLogging) {
```

Comment:
I think we shouldn't log anything here if we just started watching.

Reply:
I think it's useful. There is no indication otherwise that file system watching is enabled.

Reply:
Removed anyway, see #14342.